### PR TITLE
Expose pagination headers to /api/*

### PR DESCRIPTION
**Pull request by YodaLightsabr**
https://hackclub.slack.com/archives/C047Y01MHJQ/p1672575369699689

Pagination headers aren't currently exposed to the `fetch` API on other hosts. This adds them to `/api/v3/*`.

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -17,4 +17,12 @@
              headers: :any,
              methods: %i[get post put patch delete options head]
   end
+
+  allow do
+    resource "/api/v3/*",
```
Comment (garyhtou) [inline config/initializers/cors.rb]: maybe we can do
```suggestion
    resource "/api/*",
```

**Comment by garyhtou** [comment]
fyi, got this error in the logs
```
config/initializers/cors.rb:26: syntax error, unexpected end-of-input, expecting `end'
```

**Comment by YodaLightsabr** [comment]
image

It seems to be working now @garyhtou

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -13,6 +13,11 @@
       end
     end
 
+    resource "/api/*",
+             headers: :any,
+             methods: %i[get post put patch delete options head],
```
Comment (garyhtou) [inline config/initializers/cors.rb]: i wonder if this is necessary

**Review by garyhtou — APPROVED** [review]
perfect! good to merge as is!

File: config/initializers/cors.rb (hunk 2 above)
Comment (YodaLightsabr) [reply config/initializers/cors.rb]: I can revisit this after Bank Wrapped is done and try that out.

**Pull request merged**